test-framework/core/src/main/java/org/keycloak/testframework/cache/InfinispanLocalCache.java

```diff
@@ -0,0 +1,20 @@
+package org.keycloak.testframework.cache;
+
+public class InfinispanLocalCache implements InfinispanCache{
```

Comment:
`ClusteredKeycloakServerSupplier` uses cache type "ispn". Should we also created a `InfinispanCache` implementation for that?

The impl would be basically the same as `InfinispanLocalCache` but with `getCacheName` changed. Can we add no-op default methods to the `InfinispanCache` interface.